src/ui/castlekeysmouse.pas

```diff
@@ -661,27 +661,30 @@ function StrToKey(const S: string; const DefaultKey: TKey): TKey;
     EventType: TInputPressReleaseType;
 
     { When EventType is itKey, this is the key pressed or released.
-      Either Key <> keyNone or KeyCharacter <> #0 in this case.
-      When EventType <> itKey, then Key = keyNone and KeyCharacter = #0.
+      Either Key <> keyNone or KeyString <> '' in this case.
+      When EventType <> itKey, then Key = keyNone and KeyString = ''.
 
-      Both Key and KeyCharacter represent the same action. Sometimes one,
+      Both Key and KeyString represent the same action. Sometimes one,
       sometimes the other is useful.
 
       @bold(Not all key presses can be represented as TKey value.)
       For example, pressing '(' (opening parenthesis), which is done on most
       keyboards by pressing shift + zero, does not have any TKey value.
-      So it will generate event with Key = keyNone, but KeyCharacter = '('.
+      So it will generate event with Key = keyNone, but KeyString = '('.
 
       @bold(Likewise, not all key presses can be represented as char value.)
       For example "up arrow" (Key = keyUp) doesn't have a char code
-      (it will have KeyCharacter = #0).
```

Comment:
Instead of

```
       @bold(Likewise, not all key presses can be represented as char value.)
```

maybe better this:

```
       @bold(Likewise, not all key presses can be represented as a string or character.)
```

Reply:
Done.